rocky/katalogus/templates/boefjes.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layouts/base.html" %}
+
+{% load i18n %}
+{% load static %}
+
+{% block content %}
+    {% include "header.html" %}
+
+    <main id="main-content" tabindex="-1" class="katalogus">
+        <section>
+            {% include "partials/katalogus_header.html" %}
+            {% include "partials/plugins_navigation.html" with active="boefjes" %}
+
+            <div class="fifty-fifty">
+                <div>
+                    <h2>Boefjes</h2>
+                    <p>
+                        {% blocktranslate %}
+                            Boefjes are used to scan for objects. They detect vulnerabilities, security issues, and
+                            to give insight. Each Boefje is a separate scan that can run on a selection of objects.
+                        {% endblocktranslate %}
+                    </p>
+                    <p class="emphasized">
+                        <strong>{{ object_list|length }}</strong> Boefjes {% translate "available" %}
```

Comment:
It would be nice to pluralize *boefjes* using the `blocktranslate` and `plural` tags